dasharo-stability/capsule-update.robot

```diff
@@ -248,26 +270,47 @@ Upload Required Files
-    Send File To DUT    ./dl-cache/edk2/${file_name}_wrong_cert.cap    /capsule_testing/wrong_cert.cap
-    Log To Console    Sending ./dl-cache/edk2/${file_name}_invalid_guid.cap
-    Send File To DUT    ./dl-cache/edk2/${file_name}_invalid_guid.cap    /capsule_testing/invalid_guid.cap
-
-    # Move the directory to ESP partition so the tests work even if root
-    # file-system is part of LVM
-    Execute Command In Terminal    rm -r /boot/efi/capsule_testing
-    Execute Command In Terminal    mv /capsule_testing /boot/efi
+
+    IF    ${TESTS_IN_UBUNTU_SUPPORT}
+        Go To Ubuntu Prompt
+        # Send File To DUT uses regular user, so prepare target directory in as root
+        Execute Command In Terminal    rm -r /capsule_testing
+        Execute Command In Terminal    mkdir /capsule_testing
+        Execute Command In Terminal    chmod 777 /capsule_testing
+
+        Log To Console    Sending ./dasharo-stability/capsule-update-files/CapsuleApp.efi
+        Send File To DUT    ./dasharo-stability/capsule-update-files/CapsuleApp.efi    /capsule_testing/CapsuleApp.efi
+        Log To Console    Sending ${CAPSULE_FW_FILE}
+        Send File To DUT    ${CAPSULE_FW_FILE}    /capsule_testing/valid_capsule.cap
+        Log To Console    Sending ./dl-cache/edk2/${file_name}_wrong_cert.cap
+        Send File To DUT    ./dl-cache/edk2/${file_name}_wrong_cert.cap    /capsule_testing/wrong_cert.cap
+        Log To Console    Sending ./dl-cache/edk2/${file_name}_invalid_guid.cap
+        Send File To DUT    ./dl-cache/edk2/${file_name}_invalid_guid.cap    /capsule_testing/invalid_guid.cap
+
+        # Move the directory to ESP partition so the tests work even if root
+        # file-system is part of LVM
+        Execute Command In Terminal    rm -r /boot/efi/capsule_testing
+        Execute Command In Terminal    mv /capsule_testing /boot/efi
+    ELSE IF    ${TESTS_IN_WINDOWS_SUPPORT}
+        Go To Windows Prompt
+        Set Prompt    C:\>
+        Execute Command In Terminal    cd \
+        Execute Command In Terminal    rmdir capsule_testing
+        Execute Command In Terminal    mkdir capsule_testing
+
+        Log To Console    Sending ./dasharo-stability/capsule-update-files/CapsuleApp.efi
+        SSHLibrary.Put File
+        ...    ./dasharo-stability/capsule-update-files/CapsuleApp.efi
+        ...    C:\\capsule_testing\\CapsuleApp.efi
+        Log To Console    Sending ${CAPSULE_FW_FILE}
+        SSHLibrary.Put File    ${CAPSULE_FW_FILE}    C:\\capsule_testing\\valid_capsule.cap
+        Log To Console    Sending ./dl-cache/edk2/${file_name}_wrong_cert.cap
+        SSHLibrary.Put File    ./dl-cache/edk2/${file_name}_wrong_cert.cap    C:\\capsule_testing\\wrong_cert.cap
+        Log To Console    Sending ./dl-cache/edk2/${file_name}_invalid_guid.cap
+        SSHLibrary.Put File    ./dl-cache/edk2/${file_name}_invalid_guid.cap    C:\\capsule_testing\\invalid_guid.cap
```

Comment:
`C:` isn't ESP, so even if UEFI shell can access NTFS, it is likely to find the capsules on ESP (put there by previous attempts with `TESTS_IN_UBUNTU_SUPPORT` enabled) first. Please use command from [this superuser answer](https://superuser.com/a/735400) to mount ESP instead. SSH login should be running with admin privileges by default, so copying files shouldn't be a problem.

Reply:
Changed in: ad35a4ddff9a57115ebfe8671c60e90895a9ffdc

Reply:
@krystian-hebel made changes to deleete upload folder after fle upload - it caused tests to sometimes run capsule update from that folder. Its a good practice to cleanup after yourself ;)
617d558be528224f86ac304d147c043ff8aba15c
[log.zip](https://github.com/user-attachments/files/17331530/log.zip)

Reply:
I must have removed it after i did some other work on this branch.,Added it again: fab37e4a9ca9cd5801ae088f732e4245a188831b

Reply:
Can you do the same for Ubuntu, please?

Reply:
Never mind, Ubuntu uses `mv` and not `cp`.